config.env.template

```diff
@@ -14,8 +14,10 @@ LOGGER_ENCODING=console                 # json,console
 LOGGER_OUTPUT_PATHS=stderr              # comma separated paths
 LOGGER_ERROR_OUTPUT_PATHS=stderr        # comma separated paths
 
-# Server configurations
+# Resty Retry and Logger Config
+RESTY_DEBUG=0                           #0 or 1
```

Comment:
Better to make it just DEBUG instead of RESTY_DEBUG so that it's useful in other cases.